tests/compas_timber/test_y_butt_joint.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from compas.data import json_load
+from compas.geometry import Vector
+
+from compas_timber.connections import YButtJoint
+from compas_timber.elements import Beam
+from compas_timber.fabrication.double_cut import DoubleCut
+from compas_timber.fabrication.jack_cut import JackRafterCut
+from compas_timber.fabrication.lap import Lap
+from compas_timber.model import TimberModel
+
+
+def test_create():
```

Comment:
please add a serialization test
``` 
json_loads(json_dumps(assembly_with_joint))
```
and then make sure all beam instances are present etc.